### PR TITLE
add shit two

### PR DESCRIPTION
**Pull request by buinam2**
hi buinam2 here, please verify this and merge

### CODE REVIEW

File: src/components/ShitTwo.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function ShitTwo() {
+    return (
+        <div style={{backgroundColor: "black", color: 'white'}}>
+            Shit Two
```
Comment (buikhacnam) [inline src/components/ShitTwo.js:6]: I like this shit

**Comment by buinam2** [comment]
buinam2 here! just write a comment

**Comment by buikhacnam** [comment]
ok buinam2, I am reviewing from buinam1

File: src/components/ShitTwo.js (hunk 1 above)
Comment (buinam2) [reply src/components/ShitTwo.js:6]: ok sir

**Pull request merged**